src/bcf/report/table_report/static_reader.rs

```diff
@@ -102,6 +106,23 @@ fn calc_rows(
         reads_wr.push(base);
     }
 
+    if max_row > max_read_depth as usize {
+        let rows: Vec<u32> = (0..max_row as u32).collect();
+        let mut rng = StdRng::seed_from_u64(42);
+        let random_rows = rows
+            .iter()
+            .choose_multiple(&mut rng, max_read_depth as usize);
```

Comment:
This should be turned into a HashSet if you want to check for containment below. Otherwise, runtime becomes quadratic.

Reply:
Changed with https://github.com/rust-bio/rust-bio-tools/pull/76/commits/e16fa389c0bffc658bd4d56b64701880648e6505.